### ISSUE TITLE
Resolve errors are swallowed

### ISSUE THREAD

**Issue by chinesedfan** [rollup/rollup-plugin-node-resolve#167]
https://github.com/rollup/rollup-plugin-node-resolve/blob/master/src/index.js#L162

Shall `err` be handled first? For example, if I `require('non-existed-pkg')`, this plugin just throws nothing.

**Comment by mecurc** [comment]
I tried it.

Console output:
```bash
$ rollup -c

src/index.js → dist/bundle.js...
(!) Unresolved dependencies
https://rollupjs.org/guide/en#warning-treating-module-as-external-dependency
nonexistentmodule (imported by src/index.js)
created dist/bundle.js in 97ms
Done in 0.60s.
```

I think is OK behavior. Modules compiled with rollup can use external dependencies that can be called in runtime later.

Can be closed now. Repen if need.

**Comment by chinesedfan** [comment]
Seems fixed by #183 and released in v4.0.1. Thanks.
